analytic_resource_plan_stock_picking/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Analytic Resource Planning - get resources",
+    "version": "10.0.1.0.0",
+    "author": "Eficent",
+    "license": 'AGPL-3',
+    "website": "www.eficent.com",
+    "description": """Fetch stock for projects""",
+    "category": "Generic Modules/Projects & Services",
+    "depends": [
+        "analytic_resource_plan",
+        "analytic_location",
+        "purchase_request",
+        "project_location"
```

Comment:
@nikul-serpentcs Please set the dependency to analytic_resource_plan_purchase_request back. The module is not ported yet but this dependency is very important.